example/run_djinni.sh

```diff
@@ -28,6 +28,8 @@ java_package="com.dropbox.textsort"
 
 gen_stamp="$temp_out/gen.stamp"
 
+objc_bridging_header="$base_dir/objc/TextSort/TextSort-Bridging-Header.h"
```

Comment:
I don't love the asymmetry of this option vs. the other generation options.  Also, it appears that it's generating 2 files, even though you're giving it the name of only one.  I'd suggest making the argument specify a directory for bridging-header output, calling it something like --objc-swift-bridging-out, for symmetry with the other "-out" options.